addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -31,6 +32,9 @@ def send_mail_test(self):
             else:
                 invalid_candidates.append(candidate)
 
+        if request:
```

Comment:
Normally we shouldn't need to test this. Is there any reason why we do this ?

Reply:
Because of this error https://runbot.odoo.com/runbot/build/52455613

Reply:
Ideally, I would say we should use `MockRequest` for these tests but it's only defined in the website module so meh.
I guess we can keep this then, unless the next person as a better idea.